ossdbtoolsservice/converters/converters.py

```diff
@@ -21,7 +21,7 @@
 
 
 def convert_str(value: str):
-    return bytearray(value.encode())
+    return bytearray(str(value).encode())
```

Comment:
passed argument is already str. Why we have done this?

Reply:
Removed it